src/config/development.json

```diff
@@ -16,8 +16,8 @@
     "grantType": "client_credentials",
     "clientId": "signing-agent",
     "callback": "/v1/auth/callback",
-    "authUrl": "https://sso-dev.pathfinder.gov.bc.ca/auth/realms/devhub/protocol/openid-connect/auth",
-    "tokenUrl": "https://sso-dev.pathfinder.gov.bc.ca/auth/realms/devhub/protocol/openid-connect/token",
-    "certsUrl": "https://sso-dev.pathfinder.gov.bc.ca/auth/realms/devhub/protocol/openid-connect/certs"
+    "authUrl": "https://sso.pathfinder.gov.bc.ca/auth/realms/devhub/protocol/openid-connect/auth",
```

Comment:
@ShellyXueHan Is the API also using `sso.pathfinder.gov.bc.ca`? If its not things will break because they're using different SSO instances.

Reply:
yup, confirmed!

Reply:
the change for this is because the only dev environment uses the env vars. there will need another fix on this later, then we could reset this file to sso-dev